05_timers/module/my_module.c

```diff
@@ -0,0 +1,93 @@
+#define DEBUG
+#define pr_fmt(fmt) "%s: " fmt,  KBUILD_MODNAME
+#include <linux/module.h>
+#include <linux/kobject.h>
+#include <linux/err.h>
+#include <linux/string.h>
+#include <linux/sysfs.h>
+#include <linux/slab.h>
+#include <linux/time.h>
+#include <linux/ktime.h>
+
+static struct kobject *my_kobj;
+
+static ssize_t relative_time_show(struct kobject *kobj,
```

Comment:
This function doesn't show anything, the naming is broken. Also not necessary but preferable to use the verb_noun naming convention.